karma.conf.js

```diff
@@ -0,0 +1,70 @@
+// Karma configuration
```

Comment:
@aka-jain should we keep the karma config file here or in `frontend/` directory?